api_spec.yaml

```diff
@@ -3454,59 +4017,13 @@ paths:
         '500':
           $ref: '#/components/responses/500'
 
-  /recipe/multitenancy/tenant/list:
-    get:
-      tags:
-        - Multitenancy Recipe
-      description: |
-        Get a list of all tenants
-
-      operationId: tenantListGET
-      parameters:
-        - $ref: '#/components/parameters/multitenancyRid'
-        - $ref: '#/components/parameters/api-key'
-        - $ref: '#/components/parameters/cdi-version'
-      responses:
-        200:
-          description: List of tenants
-
-          content:
-            application/json:
-              schema:
-                type: object
-                required:
-                  - status
-                  - tenants
-                properties:
-                  status:
-                    $ref: '#/components/schemas/statusOK'
-                  tenants:
-                    type: array
-                    items:
-                      $ref: '#/components/schemas/tenantId'
-        '400':
-          $ref: '#/components/responses/400'
-
-        '401':
-          $ref: '#/components/responses/401'
-
-        '404':
-          $ref: '#/components/responses/404'
-
-        '500':
-          $ref: '#/components/responses/500'
-
   /recipe/multitenancy/config/thirdparty:
     put:
```

Comment:
remove the additional config stuff and keep it only for apple, google workspaces and boxyhq

Reply:
Done